scripts/get_version.kts

```diff
@@ -0,0 +1,15 @@
+object GetVersion {
+
+    private const val PATTERN_VERSION = "[0-9]+.[0-9]+.[0-9]+"
```

Comment:
Isn't `.` a meta character meaning `any character`? Looks like this would also match `1a3b3` 😅 Escaping the `.` or wrapping it in `[]` will match a `.` character 😄 
```suggestion
    private const val PATTERN_VERSION = "[0-9]+\\.[0-9]+\\.[0-9]+"
```
```suggestion
    private const val PATTERN_VERSION = "[0-9]+[.][0-9]+[.][0-9]+"
```